src/App.tsx

```diff
@@ -103,14 +103,14 @@ class App extends React.Component<RouteComponentProps, State> {
   }
 
   public render() {
-    if (!this.state.election) {
+    const { election } = this.state
+    if (!election) {
       return <UploadConfig setElection={this.setElection} />
     } else {
-      const { contests } = this.state.election
       return (
         <BallotContext.Provider
           value={{
-            contests,
+            election,
```

Comment:
I like this slightly broader abstraction, as the election will definitely contain more info than just contests over time.